test/empty.js

```diff
@@ -30,6 +30,13 @@ describe('empty', function() {
     eq(R.empty([1, 2, 3]), []);
   });
 
+  it('returns empty typed array of equivalent type given typed array', function() {
+    eq(R.empty(Uint8Array.from('123')), Uint8Array.from(''));
+    eq(R.empty(Uint8Array.from('123')).constructor.name, 'Uint8Array');
```

Comment:
Not sure if this is necessary.

I only added it because `eq(R.empty(Uint8Array.from('123')), Float32Array.from(''));` was passing.

Reply:
Anything that helps prevent a possible regression seems fine to me.